Solutions/Armorblox/Analytic Rules/ArmorbloxNeedsReviewAlert.yaml

```diff
@@ -0,0 +1,41 @@
+id: 322d4765-be6b-4868-9e3f-138a4f339dd6
+name: Armorblox Needs Review Alert
+description: |
+  'This rule generates an alert for an Armorblox incident where the remediation action is "Needs Review".'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Armorblox
+    dataTypes:
+      - Armorblox_CL
+queryFrequency: 10m
+queryPeriod: 10m
+triggerOperator: GreaterThan
+triggerThreshold: 0
+query: "Armorblox_CL\r\n| where remediation_actions_s contains \"Needs Review\""
```

Comment:
Please remove the quotes at the start and the end. This is not needed. Also remove the \r\n that probably is not needed.
The \ also seems to be not required. Would you please remove that as well? Thanks

Reply:
This has been done